src/cr/cube/cubepart.py

```diff
@@ -1269,6 +1287,11 @@ def inserted_row_idxs(self):
         """
         return self._assembler.inserted_row_idxs
 
+    @lazyproperty
+    def has_scale_means(self):
```

Comment:
Same as above. Seems highly redundant to introduce this to to the API, when the same thing could be checked by inspecting a different property of the same object, directly by client.

Reply:
see reply above :D any suggestion is more than welcome